api/src/lib.rs

```diff
@@ -9,16 +9,16 @@ use tlfs_crdt::ArchivedSchema;
 
 pub struct Sdk(tlfs::Sdk);
 
-pub async fn create_persistent(path: &str, package: &[u8]) -> Result<Sdk> {
+pub async fn create_persistent(path: String, package: Vec<u8>) -> Result<Sdk> {
```

Comment:
So this is to be on the safe side regarding lifetimes?

Reply:
This should fix your issue for now. It's likely we'll remove support for borrowed data, because there isn't much to gain anyway. We currently do a full copy in any case.